app/assets/stylesheets/custom.scss

```diff
@@ -47,6 +47,28 @@ h3 {
   min-height: 50px;
 }
 
+#flash_error {
```

Comment:
Avoid using id selectors<br>Selector `flash_error` should be written in lowercase with hyphens